### ISSUE TITLE
Update header logo defaults

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/oui#580]
Header logo still has defaults that refer to Elastic. These should be changed to their OpenSearch counterparts:

https://github.com/opensearch-project/oui/blob/e7dc71de0c2967ead639499e28527c5f00fdb889/src/components/header/header_logo.tsx#L57-L58

**Comment by KrooshalUX** [comment]
This might be a dupe of work tracked either here  https://github.com/opensearch-project/oui/issues/110  or here https://github.com/opensearch-project/oui/issues/113

Finishing #113 unblocks #110 , btw

**Comment by BSFishy** [comment]
> This might be a dupe of work tracked either here #110 or here #113
> 
> Finishing #113 unblocks #110 , btw

I don't think this relates to any of that work, this is specifically about the defaults for the `OuiHeaderLogo` component being outdated.

**Comment by BSFishy** [comment]
> @BSFishy I'd like to work on this, do you mind if I replace it in all the icons that use `logoElastic/Elastic`?

Sounds good. This issue is for specifically changing those 2 lines in `oui/src/components/header/header_logo.tsx`. If there is some other work you think should be done, feel free to open a new issue where it can be discussed further

**Comment by andreymyssak** [comment]
> @andreymyssak this issue can be closed now right?

@ashwin-pc yes, please close this issue
